components/library/libjpeg/pkg5

```diff
@@ -0,0 +1,9 @@
+{
+    "dependencies": [
+        "system/library"
+    ],
+    "fmris": [
+        "image/library/libjpeg"
```

Comment:
This package is already provided by `meta-packages/libjpeg` component.

Reply:
Good catch. That is now a duplicate. Removed.